src/main/java/com/jalasoft/jfc/model/command/ffmpeg/CommandVideoBitRate.java

```diff
@@ -34,7 +34,7 @@ public CommandVideoBitRate(String commandValue) {
     public String command() throws CommandValueException {
         try {
             int MINIMUM_VALUE = 200;
-            if (commandValue.isEmpty()) {
+            if (commandValue.isEmpty() || Integer.parseInt(commandValue) <= MINIMUM_VALUE || commandValue.equals(" ")) {
```

Comment:
Use magick number in commandValue.equals(" ")   --> commandValue.equals(this.SPACE)